app/repositories/onlinetesting/Phase3TestRepository.scala

```diff
@@ -57,7 +57,9 @@ class Phase3TestMongoRepository(dateTime: DateTimeFactory)(implicit mongo: () =>
   override implicit val bsonHandler: BSONHandler[BSONDocument, Phase3TestGroup] = Phase3TestGroup.bsonHandler
 
   override def nextApplicationsReadyForOnlineTesting: Future[List[OnlineTestApplication]] = {
-    val query = BSONDocument("applicationStatus" -> ApplicationStatus.PHASE2_TESTS_PASSED)
+    val query = BSONDocument("applicationStatus" -> ApplicationStatus.PHASE2_TESTS_PASSED,
+      s"progress-status.${PHASE2_TESTS_COMPLETED}" -> true
```

Comment:
Shouldn't the progress be be PHASE2_TESTS_PASSED? I think that is the terminal state for phase 2 where the test has passed